index.md

```diff
@@ -9,4 +9,6 @@ Turku &hearts; Frontend is the #1 community for frontend developers and designer
 
 We organize monthly meetups, hackathons, programming competitions and other gatherings to promote developer culture. Our doors are open for everyone: beginners, curious, hobbyists, professionals and beyond.
 
-Our meetups would not be possible without our [amazing speakers](/speakers)
+Our meetups would not be possible without our [amazing speakers](/speakers).
+
+Please note that everyone on our meetups are expected act in accordance with the [Code of Conduct](/code-of-conduct) so we can build a community that is open and welcoming to everyone.
```

Comment:
Some typos here. I think it should be "everyone in our meetups is expected to act" instead of "everyone on our meetups are expected act".

Reply:
Thanks!